interop-testing/src/main/resources/META-INF/services/io.grpc.LoadBalancerProvider

```diff
@@ -0,0 +1 @@
+io.grpc.testing.integration.RpcBehaviorLoadBalancerProvider
```

Comment:
do you want to use it to register this provider to the LoadBalancerRegistry?

Reply:
Yes exactly, this file will be in the classpath and read automatically to register any LB providers in it.

Reply:
yea ok, is it possible to call register before test and de-register after? I strongly feel that our test provider should better not mix with production service providers. But I don't know how bad it is. wdyt?
??

Reply:
That was my initial plan, but @ejona86 suggested this approach. Eric, would our interop-testing project META-INF ever mingle with the real one(s)?

Reply:
This doesn't mix with our production service providers because it is part of interop-testing. I don't think we should register this per-test, because that's not at all how the test architecture works, which is also why we aren't doing that for Ivy's interop tests. The xds interop tests don't know what test they are running, so they can't at all register per-test (vs just having hacks). We'd have to add a new argument for the binary and I don't think that really provides any value.